documentation/source/users/rmg/liquids.rst

```diff
@@ -308,13 +299,90 @@ The work is in progress to implement the temperature dependent solvation free en
 Group additivity method for solute descriptor estimation
 --------------------------------------------------------
 
-Group additivity is a convenient way of estimating the thermochemistry for thousands of species sampled 
+Group additivity or group contribution is a convenient way of estimating the thermochemistry for thousands of species sampled
 in a typical mechanism generation job. Use of the Abraham Model in RMG requires a similar approach 
-to estimate the solute descriptors (`A, B, E, L,` and `S`). Platts et al. ([Platts1999]_) proposed such a scheme 
-employing a set of 81 molecular fragments for estimating `B, E, L, V` and `S` and another set of 51 fragments for 
-the estimation of `A`. These fragments are implemented in RMG but are limited to the compounds containing H, C,
-O, N, and S. The value of a given descriptor for a molecule is obtained by summing the contributions from each
-fragment found in the molecule and the intercept associated with that descriptor.
+to estimate the solute descriptors (`A, B, E, L,` and `S`). RMG uses the group additivity method devised by Chung et al. [Chung2021]_
+for solute descriptor estimations. The solute descriptor group additivity follows the same scheme as the RMG's group
```

Comment:
"estimations" => "estimation" throughout

Reply:
it's fixed in this fixup commit 2d234db.